### PR TITLE
Update code owners for SDKs

### PR DESCRIPTION
**Pull request by pallavit**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by jsquire** [comment]
@ronniegeraghty :  Do you mind reflecting these over to fabric bot when finalized?    Labels:  "Mixed Reality"  and  "Load Testing Service" exist.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -393,6 +393,12 @@
 # ServiceLabel: %Lab Services %Service Attention
 sdk/labservices/Microsoft.Azure.Management.LabServices/           @Tanmayeekamath
 
+# ServiceLabel: %LoadTest Service %Service Attention
+sdk/loadtestservice/            @abranj1219 @ninallam @NiveditJain
+# PRLabel: %loadtestservice
```
Comment (jsquire) [inline .github/CODEOWNERS:398]:
```suggestion

# PRLabel: %Load Test Service
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -393,6 +393,12 @@
 # ServiceLabel: %Lab Services %Service Attention
 sdk/labservices/Microsoft.Azure.Management.LabServices/           @Tanmayeekamath
 
+# ServiceLabel: %LoadTest Service %Service Attention
```
Comment (jsquire) [inline .github/CODEOWNERS:396]:
```suggestion
# ServiceLabel: %Load Test Service %Service Attention
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -468,6 +474,11 @@ sdk/monitor/ci.yml                         @nisha-bhatia @ShivangiReja
 # ServiceLabel: %MySQL %Service Attention
 /sdk/mysql/Microsoft.Azure.Management.MySQL/            @ajlam @ambhatna @kummanish
 
+# ServiceLabel: %Mixed Reality %Service Attention
+/sdk/mixedreality/                                 @crtreasu @rgarcia
+# PRLabel: %MixedReality
```
Comment (jsquire) [inline .github/CODEOWNERS:479]:
```suggestion

# PRLabel: %Mixed Reality
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -525,6 +536,11 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 # ServiceLabel: %Notification Hub %Service Attention
 /sdk/notificationhubs/Microsoft.Azure.Management.NotificationHubs/            @tjsomasundaram
 
+# ServiceLabel: %Mixed Reality %Service Attention
```
Comment (jsquire) [inline .github/CODEOWNERS:539]:
```suggestion
```

There can only be one rule for a given label, so the "ServiceLabel" bit isn't needed.

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -525,6 +536,11 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
...
+/sdk/objectanchors/                                 @crtreasu @rgarcia
+# PRLabel: %MixedReality
```
Comment (jsquire) [inline .github/CODEOWNERS:541]:
```suggestion

# PRLabel: %Mixed Reality
```

**Comment by ronniegeraghty** [comment]
> @ronniegeraghty Ronnie Geraghty FTE : Do you mind reflecting these over to fabric bot when finalized? Labels: "Mixed Reality" and "Load Testing Service" exist.

Sure thing

**Review by jsquire — APPROVED** [review]

**Comment by pallavit** [comment]
Thank you for the review :)

**Pull request merged**